app/services/bibid_updater.rb

```diff
@@ -40,6 +40,7 @@ def resources
           query_service.custom_queries.find_by_property(property: :source_metadata_identifier, value: [], lazy: true).select do |resource|
             id = resource.source_metadata_identifier.first
             next if id =~ /99.*3506421/
+            next if transform_id(id).length > 18
             RemoteRecord.bibdata?(id)
```

Comment:
why is this > 18?

Reply:
The biggest ID that started with "9999" was 16 characters.